### PR TITLE
Updates for v1.4 release

### PR DESCRIPTION
**Pull request by langner**

### CODE REVIEW

**Comment by berquist** [comment]
Is there anything else to be added here, or can I merge it?

**Comment by langner** [comment]
Updated the ANNOUNCE some more and added a DOI I prereserved with Zenodo. Should be good to go now, but let me know if there anything missing. Once this goes in I'll cherrypick on the v1.4 branch and finalize the release.

File: ANNOUNCE [hunk 1]
```diff
@@ -1,6 +1,6 @@
-On behalf of the cclib development team, we are pleased to announce the release of cclib 1.3.1, which is now available for download from http://cclib.github.io. This is a minor update to version 1.3 that includes bug fixes and small improvements.
+On behalf of the cclib development team, we are pleased to announce the release of cclib 1.4, which is now available for download from http://cclib.github.io. This is a major update to version 1.3.1 that includes new parses and attributes, asll well as many bug fixes and small improvements.
```
Comment (berquist) [inline ANNOUNCE]: small typo here

Comment (langner) [inline ANNOUNCE]: Fixed.

**Pull request merged**